File: csharp/resistor-color/ResistorColorTests.cs
using Xunit;

public class ResistorColorTests
{
    [Fact]
    public void Black()
    {
        Assert.Equal(0, ResistorColor.ColorCode("black"));
    }

    [Fact]
    public void White()
    {
        Assert.Equal(9, ResistorColor.ColorCode("white"));
    }

    [Fact]
    public void Orange()
    {
        Assert.Equal(3, ResistorColor.ColorCode("orange"));
    }

    [Fact]
    public void Colors()
    {
        Assert.Equal(new[] { "black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white" }, ResistorColor.Colors());
    }
}

File: vbnet/resistor-color/ResistorColorTests.vb
Imports Xunit
Public Class ResistorColorTest
    <Fact>
    Public Sub Black()
        Assert.Equal(0, ColorCode("black"))
    End Sub
    <Fact>
    Public Sub White()
        Assert.Equal(9, ColorCode("white"))
    End Sub
    <Fact>
    Public Sub Orange()
        Assert.Equal(3, ColorCode("orange"))
    End Sub
    <Fact>
    Public Sub Colors()
        Assert.Equal({"black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"}, ResistorColor.Colors())
    End Sub
End Class